modules/test/playwright/pages/object-web/list-type/ListTypeDefinitionsPage.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {ApplicationsMenuPage} from '../../product-navigation-applications-menu/ApplicationsMenuPage';
+
+export class ListTypeDefinitionsPage {
+	readonly addPicklistButton: Locator;
+	readonly applicationsMenuPage: ApplicationsMenuPage;
+	readonly page: Page;
+	readonly picklistNameInput: Locator;
+	readonly savePicklistButton: Locator;
+
+	constructor(page: Page) {
+		this.addPicklistButton = page
+			.getByRole('button', {
+				name: 'Add Picklist',
+			})
+			.first();
+		this.applicationsMenuPage = new ApplicationsMenuPage(page);
+		this.page = page;
+		this.picklistNameInput = page.locator(
+			'input[id="listTypeDefinitionName"]'
```

Comment:
See my other comment regarding the use of `page.locator()`.

Reply:
same thing as explained above!

Reply:
In this case, our team owns this application. It looks like [input is not bound to a label](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label). Can you check this? Then you can add a `for` attribute to `label` pointing to this `id`. This would be low-effort and fix an hidden accessibility problem. See my [comment above](https://github.com/liferay-objects/liferay-portal/pull/3353#discussion_r1606756017) for more context.

Reply:
thanks! the code already had it, so i just needed to use `getByLabel` :blush: